packages/framework-integration-tests/integration/providers/kubernetes/nuke/setup.ts

```diff
@@ -0,0 +1,23 @@
+import { overrideWithBoosterLocalDependencies } from '../../../helper/deps-helper'
+import { nuke } from '../deploy'
+import { createSandboxProject, removeSandboxProject } from '../../../../../cli/src/common/sandbox'
+import { sandboxPathFor } from '../../../helper/file-helper'
+import { sandboxProjectName } from '../constants'
+import { exec } from 'child-process-promise'
+import { createDaprComponentFile } from '../util'
+
+before(async () => {
+  const sandboxPath = sandboxPathFor(sandboxProjectName)
+  const configuredAssets = ['assets', 'assetFile.txt']
+  const sandboxedProject = createSandboxProject(sandboxPath, configuredAssets)
+
+  await overrideWithBoosterLocalDependencies(sandboxPath)
+  // Only the deploy command creates the production dependencies
+  await exec('npm install --production --no-bin-links', { cwd: sandboxPath })
```

Comment:
This is not needed. We were getting an error if we didn't include this with npm version < 7, but with npm 7+, this is unneeded. 

You can safely remove it (check the nuke setup test in AWS, it didn't have it anymore)

Reply:
Hey @alvaroloes the problem with this is that the deployment removed the `.deploy` folder, and the nuke doesn't generate a new one. Also the nuke command is not looking for a `.deploy` folder, but for the root path of the application (not sure if it's a bug or it's expected): https://github.com/boostercloud/booster/blob/main/packages/cli/src/commands/nuke.ts#L61